toqito/channel_ops/tests/test_kraus_to_choi.py

```diff
@@ -128,3 +128,9 @@ def test_kraus_to_choi(test_input, expected):
     """Test function works as expected for valid inputs."""
     calculated = kraus_to_choi(test_input)
     assert np.isclose(calculated, expected).all()
+
+
```

Comment:
We should add some tests where the `sys` argument supplied is valid (i.e. non-negative).